lib/winapi/profiling.c

```diff
@@ -0,0 +1,20 @@
+#include <profileapi.h>
+#include <assert.h>
+#include <stddef.h>
+#include <xboxkrnl/xboxkrnl.h>
+
+BOOL QueryPerformanceCounter (LARGE_INTEGER *lpPerformanceCount)
+{
+    assert(lpPerformanceCount != NULL);
+
+    lpPerformanceCount->QuadPart = KeQueryPerformanceCounter();
+    return 1;
```

Comment:
Why not use the `BOOL` specific values like `TRUE`?

Reply:
An oversight from copying to code over from another branch. It uses `TRUE` now.